### PR TITLE
docs: refactor client-control

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update document under `docs/zh/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/client-control.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ---
 title: client-control
+keywords:
+  - APISIX
+  - API 网关
+  - Client Control
+description: 本文介绍了 Apache APISIX proxy-control 插件的相关操作，你可以使用此插件动态地控制 Nginx 处理客户端的请求的行为。
```
Comment (hf400159) [inline docs/zh/latest/plugins/client-control.md:7]:
```suggestion
description: 本文介绍了 Apache APISIX proxy-control 插件的相关操作，你可以使用此插件动态地控制 NGINX 处理客户端的请求的行为。
```

File: docs/en/latest/plugins/client-control.md [hunk 2]
```diff
@@ -2,10 +2,9 @@
 title: client-control
 keywords:
   - APISIX
-  - Plugin
+  - API Gateway
   - Client Control
-  - client-control
-description: This document contains information about the Apache APISIX client-control Plugin.
+description: This document contains information about the Apache APISIX client-control Plugin, you can use it to  control the behavior of Nginx to handle a client request dynamically.
```
Comment (SylviaBABY) [inline docs/en/latest/plugins/client-control.md:7]:
```suggestion
description: This document describes the Apache APISIX client-control Plugin, you can use it to control NGINX behavior to handle a client request dynamically.
```

File: docs/zh/latest/plugins/client-control.md [hunk 3]
```diff
@@ -25,17 +30,21 @@ title: client-control
 
 `client-control` 插件能够动态地控制 Nginx 处理客户端的请求的行为。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/client-control.md:31]:
```suggestion
`client-control` 插件能够动态地控制 NGINX 处理客户端的请求行为。
```

File: docs/zh/latest/plugins/client-control.md [hunk 4]
```diff
@@ -23,22 +28,27 @@ title: client-control
 
 ## 描述
 
-`client-control` 插件能够动态地控制 Nginx 处理客户端的请求的行为。
+`client-control` 插件能够动态地控制 NGINX 处理客户端的请求的行为。
```
Comment (hf400159) [inline docs/zh/latest/plugins/client-control.md:31]: Can you describe it more clearly? For example, by setting the size of the request body to control client access? The current description is too broad.

File: docs/zh/latest/plugins/client-control.md [hunk 5]
```diff
@@ -57,11 +67,15 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f03433
...
 ## 测试插件
 
-使用 `curl` 去测试：
+启用插件后，使用 `curl` 命令请求该路由：
 
 ```shell
 curl -i http://127.0.0.1:9080/index.html -d '123'
+```
 
+在配置插件时设置了 `max_body_size` 为 `1`，如果返回的 HTTP 响应头中带有 `413` 状态码，则表示插件生效：
```
Comment (hf400159) [inline docs/zh/latest/plugins/client-control.md:76]:
```suggestion
因为在配置插件时设置了 `max_body_size` 为 `1`，所以返回的 HTTP 响应头中如果带有 `413` 状态码，则表示插件生效：
```

File: docs/zh/latest/plugins/client-control.md [hunk 6]
```diff
@@ -75,7 +89,7 @@ HTTP/1.1 413 Request Entity Too Large
 
 ## 禁用插件
 
-当您要禁用 `client-control` 插件时，这很简单，您可以在插件配置中删除相应的 json 配置，无需重新启动服务，它将立即生效：
+当你需要禁用该插件时，可以通过以下命令删除相应的 JSON 配置，APISIX 将会自动重新加载相关配置，无需重启服务：
 
 ```shell
 curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (hf400159) [inline docs/zh/latest/plugins/client-control.md:95]:
```suggestion
curl http://127.0.0.1:9080/apisix/admin/routes/1  \
-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**